library/Freckle/App/Stats.hs

```diff
@@ -15,7 +15,7 @@ module Freckle.App.Stats
   , tagsL
   , withStatsClient
   , HasStatsClient(..)
-
+  , withGauge
```

Comment:
I might put this at the end of the _Reporting_ section.

Reply:
I put it next to HasStatsClient because they're used together. If there's a strong preference then I'll move it.

Reply:
It's not a strong preference, no, but I am curious about the reasoning: aren't all the reporting functions "used together" with `HasStatsClient`?